en/docs/install-and-setup/prerequisites.md

```diff
@@ -119,6 +119,32 @@ Listed below are the prerequisites for a successful deployment:
    </tbody>
 </table>
 
+### Network requirements
```

Comment:
i feel this is not a pre-requestie,but a seperate section as production deployment guidelines